### ISSUE TITLE
Something wrong with OpenAI's Large V2 model?

### ISSUE THREAD

**Issue by ggerganov** [ggml-org/whisper.cpp#675]
So, I haven't looked in details, but I suspect there might be something wrong in the new `large` model released by OpenAI. Keep in mind this is very anecdotal evidence atm, so I might be completely wrong.

Running the `main` example with enabled color coding for the token probabilities `-pc` we normally get the following results:

- `base.en`

![image](https://user-images.githubusercontent.com/1991296/228174981-ba5d3715-9823-49fc-b292-9bfe2ee13dd7.png)

- `small.en`

![image](https://user-images.githubusercontent.com/1991296/228174120-948312cb-c564-40aa-8375-896ceba88297.png)

- `small`

![image](https://user-images.githubusercontent.com/1991296/228174787-db56c630-e2be-445a-88ec-334084c65acd.png)

- `medium`

![image](https://user-images.githubusercontent.com/1991296/228174875-7c30af48-c504-49c0-93be-eccc3a89d225.png)

- `medium.en`

![image](https://user-images.githubusercontent.com/1991296/228174388-f536a88d-7ac5-43f3-bd0c-3ce146a838b0.png)

However, this is what the color coding look like when using the new `large` model (i.e. v2):

- `large`

![image](https://user-images.githubusercontent.com/1991296/228174558-5eda0103-969b-45e8-b0c0-20f0404dd8fb.png)

As a comparison, this is the same run, but using the old `large` model - i.e. v1:

- `large-v1`

![image](https://user-images.githubusercontent.com/1991296/228174660-0e5d2659-aa49-4c48-939e-8b38bac4efa0.png)

So somehow the logits with `v2` seem to be all over the place which is not observed for any of the other models.
Still, I need to double check all these observations, but I think there is something not quite right with `large-v2`

**Comment by Alumniminium** [comment]
Please stop calling it v2. There's an actual v2 now.
https://huggingface.co/openai/whisper-large-v2

**Comment by bayesian** [comment]
large_v2 is trained with BPE dropout, so its target tokens are a stochastic distribution. That might explain smoother logits from large_v2
